app/src/components/annotation/AnnotationInputExplanation.tsx

```diff
@@ -39,9 +39,10 @@ export const AnnotationInputExplanation = ({
         className="annotation-input-explanation"
         css={css`
           position: absolute;
-          top: 0;
+          top: 6px;
           right: 4px;
           width: ${EXPLANATION_LABEL_WIDTH};
+          font-size: var(--ac-global-dimension-static-font-size-75);
```

Comment:
<img width="559" alt="Screenshot 2025-05-06 at 9 51 51 AM" src="https://github.com/user-attachments/assets/3e54194d-ab07-46dd-b4a5-7621b0e949fa" />
first setting of the annotation doesn't light the explanation link up.

Reply:
I cannot replicate this